google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -175,6 +178,14 @@ def run(self):
                 self._store_resource(None, last_res_type)
                 self.session.flush()
 
+                for (policy, resource) in inventory.iter(
```

Comment:
the dataset policy should be treated similar to iam policy. in _store_dataset_policy, the dataset policy can be stored to resource table. When we decide how to deal with dataset policy, the changes will be limited to _store_dataset_policy